common/settings.h

```diff
@@ -22,6 +22,14 @@
 
 namespace flutter {
 
+// The combination of targeted graphics API and Impeller support.
+enum class AndroidRenderingAPI {
```

Comment:
"Android"RenderingAPI going in common is unfortunate. I suppose no more so that common itself. Couldn't this go in `//flutter/shell/common` instead?

Reply:
Not unless common/settings.h can import flutter/common.